src/assets/base.css

```
@@ -0,0 +1,46 @@
+.center {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+}
+
+.flex-container{
+    display: flex;
+    padding: 10px;
+}
+.hidden {
+    display: none;
+}
+
+.smooth-scroll {
+    scroll-behavior: smooth;
+}
+
+.no-scrollbar {
+    overflow: hidden;
+}
+.scrollbar {
+    overflow: show;
+}
+
+.dotted-border {
```

We could try to follow BEM naming patterns for CSS.

```suggestion
.border--dotted {
```